CIP-0072/README.md

```diff
@@ -83,12 +77,10 @@ The on-chain dApp registration certificate MUST follow canonical JSON and be ser
   - *`REGISTER`*: The certificate is asserting that the dApp is registered for the first time or is providing an update.
   - *`DE_REGISTER`*: The certificate is asserting that the dApp is deprecated / archived. So, no further dApp's on-chain update is expected.
 
-*`rootHash`*: The hash of the entire offchain metadata tree object. This hash is used by clients to verify the integrity of the metadata tree object. When reading a metadata tree object, the client should calculate the hash of the object and compare it with the `rootHash` property. If the two hashes don't match, the client should discard the object. The metadata tree object is a JSON object that contains the dApp's metadata. The metadata tree object is described in the next section. Please note that off-chain JSON must be converted into RFC 8765 canonical form before taking the hash!
+*`rootHash`*: The blake2b-256 hash of the entire offchain metadata tree object. This hash is used by clients to verify the integrity of the metadata tree object. When reading a metadata tree object, the client should calculate the hash of the object and compare it with the `rootHash` property. If the two hashes don't match, the client should discard the object. The metadata tree object is a JSON object that contains the dApp's metadata. The metadata tree object is described in the next section. Please note that off-chain JSON must be converted into RFC 8765 canonical form before taking the hash!
 
 *`metadata`*: An array of links to the dApp's metadata. The metadata is a JSON compatible RFC 8785 object that contains the dApp's metadata.
 
-*`signature`*: The signature of the certificate. The publishers generate the signature is by first turning on-chain JSON into a canonical form (RFC 8765), hashing it with blake2b-256 and generating a signature of the hash. Stores / clients can verify the signature by repeating the process, they can use the public key to verify the signature of the certificate. Fields used for canonical JSON: ["subject", "rootHash", "metadata","type"]. Please note that a signature should be generated of blake2b-256 hash as a byte array, not as a hex represented string(!).
-
 ### On-chain Schemas
 
 [On-chain CDDL for registration / de-registration (Version 1)](./version_1_onchain.cddl)
```

Comment:
```suggestion
[On-chain CDDL for registration / de-registration (Version 1.0.0)](./version_1_onchain.cddl)
```
If we are changing the versioning schema, we should probably also align the naming on the supplied schemas

Reply:
Suggestion applied, additionally I also renamed the files following semantic version pattern.

Reply:
@danielmain it came up in our meeting yesterday that semantic versioning is generally undesirable relative to something that can be stored as an integer as is easiest for blockchain data.  Do the 2nd & 3rd semantic version numbers provide that much utility vs. a single integer that we couldn't simply use an integer as for all other CIP data structures?  cc @Ryun1 re: https://github.com/cardano-foundation/CIPs/pull/612#issuecomment-1808357567

Reply:
@rphair is not only a standard in the software development, the reasoning of semver is to represent small backward-compatible fixes against new improvements and new versions that bring breaking-changes implicitly.

> something that can be stored as an integer as is easiest for blockchain data
We are not storing on the blockchain the version 🤔 can you please elaborate more?

Reply:
@danielmain given that we all understand the principle & purpose of semantic versioning, I'm just passing on the response from the meeting.

In general, just because the version number is stored off-chain doesn't mean it can't be referenced by on-chain data & may have to in certain dApp or transaction metadata... then you have the usual problems of data type & byte ordering for the version string or component integers.

Reply:
Anyway @danielmain if semver is what you believe is best, of course we'll do it that way.  This reservation was expressed at the meeting so I'm just recording it here... I'll leave this open for now in case @Ryun1 @Crypto2099 want to comment.

Reply:
> Anyway @danielmain if semver is what you believe is best, of course we'll do it that way. This reservation was expressed at the meeting so I'm just recording it here... I'll leave this open for now in case @Ryun1 @Crypto2099 want to comment.

Thank you @rphair for these valid observations 🙇🏻 
I will do my best to be in the next meeting, so we can solve questions much more quickly.

Reply:
thanks @danielmain - there are not too many new items so I've put this at the end of the Review agenda for the next CIP meeting ([28 November](https://hackmd.io/@cip-editors/77)) after we get through some other planned discussions.  With your updates in the last period I believe there is more to discuss, especially if you are present. 😎